README.md

```diff
@@ -3,230 +3,161 @@
+### 3. List projects
 
-```bash
-$ cd deploy
-$ kubectl apply -f . -n <project-name>
-# verify stateful store
-$ kubectl get -n <project-name> statefulstore
-NAME                  AGE
-shopping-store   21m
-# verify stateful services
-$ kubectl -n <project-name>  get statefulservices
-NAME            AGE    REPLICAS   STATUS
-shopping-cart   7m     1          Running
-frontend        4m     1          Running
+```shell
+$ csctl projects list
 ```
 
-`https://<project-name>.us-east1.apps.lbcs.dev/pages/index.html`
+You should see the project listed:
 
-## Building and deploying the Sample application
+```shell
+  NAME                   DESCRIPTION            STATUS   ID
+  sample-shopping-cart   Shopping Cart Sample   active   39ad1d96-466a-4d07-b826-b30509bda21b
+```
 
-### Frontend Service
+Wait until you get an email about your project being created and approved!
 
-The `frontend` service is a frontend web application written in TypeScript. It is backed by a `stateless` service that will serve the compiled JavaScript, html and images. This service makes `grpc-web` calls directly to the other services to get the data that it needs.
+### 4. Set the current project
 
-#### Getting container image ready
+```shell
+$ export CSCTL_CURRENT_PROJECT=`csctl projects list -o json | \
+  jq -r '.[] | select(.friendly_name == "sample-shopping-cart") | .name / "/" | .[1]'`
+$ csctl config set project $CSCTL_CURRENT_PROJECT
+```
 
-You can use the pre-built `lightbend-docker-registry.bintray.io/cloudstate-samples/frontend:latest` container image available at Lightbend Cloudstate samples repository.
+### 5. Deploy the frontend
 
-Alternatively, you can clone the [cloudstateio/samples-ui-shoppingcart](https://github.com/cloudstateio/samples-ui-shoppingcart) repository and follow the instructions there to build an image and deploy it to your own container image repository.
+```shell
+$ csctl deploy frontend lightbend-docker-registry.bintray.io/cloudstate-samples/frontend
+```
 
-#### Deploying the frontend service
+### 6. Create the store
 
-Change into the `deploy` folder
-```
-$ cd deploy
-```
-If you have built your own container image, edit the `frontend.yaml` to point to the image that you just pushed.
-```
-$ cat frontend.yaml
-apiVersion: cloudstate.io/v1alpha1
-kind: StatefulService
-metadata:
-  name: frontend
-spec:
-  containers:
-  - image: lightbend-docker-registry.bintray.io/cloudstate-samples/shopping-cart:latest # <-- Change this to your repo/image
-    name: frontend
+```shell
+$ csctl store deploy shopping-store
 ```
 
-Deploy the service to your project namespace
-```
-$ kubectl apply -f frontend.yaml -n <project_name>
-statefulservice.cloudstate.io/frontend created
-```
+Wait for the store to be created
 
+```shell
+$ watch csctl stores get
+```
 
-### Stateful Store
+Proceed when `STATUS` is `ready`. This will take approximately 5 minutes.
```

Comment:
I always hesitate to put a time since I suspect it might vary. How about, Proceed when the project `STATUS` is `ready`, this can take some time.